### PR TITLE
FAB-4088 Milestone 1 & 2

### PR DESCRIPTION
**Pull request by mcriscolo-cs**

### CODE REVIEW

File: profiles-jar-examples/.gradle/buildOutputCleanup/cache.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#Mon Jun 06 03:35:26 CDT 2022
+gradle.version=7.4
```
Comment (laguirre-cs) [inline profiles-jar-examples/.gradle/buildOutputCleanup/cache.properties:2]: Need to remove the `.gradle/` dir - guess it was added before the .gitignore took place ? (I can include this when porting over my changes)

Comment (mcriscolo-cs) [reply profiles-jar-examples/.gradle/buildOutputCleanup/cache.properties:2]: yeah..oops

File: profiles-jar-examples/main-app/src/main/java/com/c12e/cortex/examples/BaseCommand.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        defaults.put("spark.hadoop.fs.s3a.multipart.threshold", "512M");
+        defaults.put("spark.hadoop.fs.s3a.fast.upload.active.blocks", "2048");
+        defaults.put("spark.hadoop.fs.s3a.committer.threads", "2048");
+        defaults.put("spark.hadoop.fs.s3a.max.total.tasks", "2048");
+        defaults.put("spark.hadoop.fs.s3a.threads.max", "2048");
+        defaults.put("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension");
+        defaults.put("spark.sql.catalog.spark_catalog", "org.apache.spark.sql.delta.catalog.DeltaCatalog");
+        defaults.put("spark.databricks.delta.schema.autoMerge.enabled", "true");
+        defaults.put("spark.databricks.delta.merge.repartitionBeforeWrite.enabled", "true");
+        defaults.put("spark.kubernetes.local.dirs.tmpfs", "true");
```
Comment (bpandey-CS) [inline profiles-jar-examples/main-app/src/main/java/com/c12e/cortex/examples/BaseCommand.java:62]: we should not have this as default

**Comment by bpandey-CS** [comment]
Should we add tests to the build pipeline?

**Comment by laguirre-cs** [comment]
Closing this. Contents have been moved to: https://github.com/CognitiveScale/cortex-profiles-sdk-examples

**Pull request closed without merging**